tasks.c

```diff
@@ -6271,7 +6270,7 @@ static void prvCheckTasksWaitingTermination( void )
                 /* MISRA Ref 11.5.3 [Void pointer assignment] */
                 /* More details at: https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/MISRA.md#rule-115 */
                 /* coverity[misra_c_2012_rule_11_5_violation] */
-                pxTCB = listGET_LIST_ITEM_OWNER( pxIterator );
+                TCB_t * pxTCB = listGET_LIST_ITEM_OWNER( pxIterator );
```

Comment:
Can we have the declaration as top as earlier without volatile? It would resolve C90: Wno-declaration-after-statement